components/bl-collapsiblePanel-component/src/index.js

```diff
@@ -0,0 +1,64 @@
+import { useCallback, useState } from 'react';
+
+const { cn } = BackendlessUI.CSSUtils;
+
+export default function CollapsiblePanelComponent({ component, elRef, eventHandlers, pods }) {
+  const { classList, style, display, title } = component;
+  const { onOpen, onClose } = eventHandlers;
+
+  const [isActive, setIsActive] = useState(false);
```

Comment:
`active` in the context of a collapsable panel doesn't sound correct 

would be better to have `expanded`

```
 const [expanded, setExpanded] = useState(false);
```